doc/api/cli.md

```diff
@@ -1385,6 +1385,28 @@ threadpool by setting the `'UV_THREADPOOL_SIZE'` environment variable to a value
 greater than `4` (its current default value). For more information, see the
 [libuv threadpool documentation][].
 
+## Useful V8 options
+
+V8 has its own set of CLI options. Any V8 CLI option that is provided to `node` will
+be passed on to V8 to handle. V8's options have _no stability guarantee_.
+The V8 team themselves don't consider them to be part of their formal API,
+and reserve the right to change them at any time. The Node.js team does not
+consider them covered by the Node.js stability guarantees. Many of the V8
+options are of interest only to V8 developers. Despite this, there is a small
+set of V8 options that are widely applicable to Node.js, and they are
+documented here:
+
+### `--max-old-space-size=SIZE` (in Mbytes)
+
+Sets the max memory size of V8's old memory section. As memory
+consumption approaches the limit, V8 will spend more time on
+garbage collection in an effort to free unused memory.
+
+On a machine with 2GB of memory it is suggested to set this to
+1536 (1.5GB) to leave some memory for other uses and avoid swapping.
+
+E.g. `node --max-old-space-size=1536 index.js`
```

Comment:
Optional nit: Delete this line.

Reply:
I personally feel an example helps a lot when introducing to a flag. It clearly presents how to pass an argument.

Reply:
> I personally feel an example helps a lot when being introduced to a flag

If we want to provide an example, it should conform with the formatting of the other examples provided elsewhere in the doc. Otherwise, each entry is inventing its own example formatting, and we definitely do not want that. The existing format is more useful than this as it provides context, such as what the anticipated output might look like. So, this example might show the error one might get if running without the flag, then show the command succeeding with the flag.

(That said, most of our entries do not contain examples and that seems appropriate in this document.)

Reply:
Totally agree with your point, Can you point me to some of the correct existing formats?
What I see a mixture of formats, some given below
1. node --name-of-flag index.mjs
2. node --name-of-flag <file>

Reply:
I don't mean the format of the command itself. I mean the markdown formatting that denotes an example.

````markdown
```console
$ node --cpu-prof index.js
$ ls *.cpuprofile
CPU.20190409.202950.15293.0.0.cpuprofile
```
````

Reply:
@Trott checkout the formatting once now.